framework/src/components/cache/cache.js

```diff
@@ -43,26 +43,35 @@ class Cache {
 		// TOFIX: app crashes with FTL error when launchin app with CACHE_ENABLE=true
 		// but cache server is not available.
 		this.client = redis.createClient(this.options);
-		this.client.on('error', this._onConnectionError.bind(this));
-		this.client.once('ready', this._onReady.bind(this));
-	}
 
-	_onConnectionError(err) {
-		// Called if the "error" event occured before "ready" event
-		this.logger.warn('App was unable to connect to Cache server', err);
+		return new Promise((resolve, reject) => {
```

Comment:
What was the problem here?

Reply:
Ahh.. that's the famous cache test failure fix... 

We were trying to run ever tests even before Redis client is connected to Redis server.